### PR TITLE
[KIECLOUD-218] Logs are constantly flooded with 'Unable to load key store. Using password from configuration'

### PR DESCRIPTION
**Pull request by errantepiphany**
[KIECLOUD-218] Logs are constantly flooded with 'Unable to load key store. Using password from configuration'
https://issues.jboss.org/browse/KIECLOUD-218

Signed-off-by: David Ward <dward@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: jboss-kie-workbench/added/launch/jboss-kie-workbench.sh [hunk 1]
```diff
@@ -157,8 +157,19 @@ function configure_guvnor_settings() {
...
         JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dbuild.enable-incremental=${buildEnableIncremental}"
     fi
-    # see scripts/jboss-kie-wildfly-common/configure.sh
-    JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.jbpm.designer.perspective=full -Ddesignerdataobjects=false"
+    if [[ $JBOSS_PRODUCT =~ rhpam\-businesscentral(\-monitoring)? ]]; then
+        local jbpmKeystore="${JBOSS_HOME}/standalone/configuration/jBPMKeystore.jceks"
+        if [ -f "${jbpmKeystore}" ]; then
+            # KIECLOUD-218, JBPM-8400
+            JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dkie.keystore.keyStoreURL=file://${jbpmKeystore}"
+            JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dkie.keystore.keyStorePwd=jBPMKeyStorePassword"
```
Comment (spolti) [inline jboss-kie-workbench/added/launch/jboss-kie-workbench.sh]: Is expected these values be hardcoded?

Comment (errantepiphany) [reply jboss-kie-workbench/added/launch/jboss-kie-workbench.sh]: It is here: https://github.com/kiegroup/kie-wb-distributions/pull/911/files#diff-69e71f5de9b5efa55b63d55aecfe13c9R52
The main point of that PR was just to silence the warnings...

However, I'm trying to better understand the purpose of these properties. You can search for them here: https://docs.jboss.org/jbpm/release/latest/jbpm-docs/html_single/
Perhaps we can just generate the keystore using the controller username and password that we already know (set via template params). I am labeling this PR as "wip" until I figure it out.

Comment (errantepiphany) [reply jboss-kie-workbench/added/launch/jboss-kie-workbench.sh]: @spolti The hardcoding to access the truststore and key is not something that I could see a workaround for, however the encrypted value of the kie server and kie controller passwords stored inside are now dynamically set to the correct values.

Comment (spolti) [reply jboss-kie-workbench/added/launch/jboss-kie-workbench.sh]: ok, thanks for the explanation.

**Pull request merged**